content/en/docs/instrumentation/php/manual.md

```diff
@@ -120,7 +120,7 @@ Most of the time, we want to correlate
 [spans](/docs/concepts/signals/traces/#spans-in-opentelemetry) for nested
 operations. OpenTelemetry supports tracing within processes and across remote
 processes. For more details how to share context between remote processes, see
-[Context Propagation](#context-propagation).
+[Context Propagation](./propagation.md).
```

Comment:
```suggestion
[Context Propagation](/docs/instrumentation/php/propagation).
```

Reply:
thanks, I'm slowly learning how to run all the checks via gitpod. have updated to use the link format you provided...